lib/sidekiq_worker_loader.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+require "sidekiq"
+require "sidekiq/api"
+require "sidekiq/testing"
+require "rspec-sidekiq"
+require "sidekiq-poison-pill-remedy"
+require_relative "jobs/my_job"
```

Comment:
given that this is only for testing, please don't include this on a gem level, but rather on a spec_helper level